### PR TITLE
📖  Add capz identity instructions to quick start

### PR DESCRIPTION
**Pull request by nader-ziada**
**What this PR does / why we need it**:
- add instructions to create secret for identity password
- remove AzureEnvironment field as its now moved to AzureCluster and defaulted

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

/cc @CecileRobertMichon

### CODE REVIEW

**Review by vincepri — APPROVED** [review]
/lgtm
/assign @CecileRobertMichon

File: docs/book/src/user/quick-start.md [hunk 1]
```diff
@@ -210,15 +210,20 @@ export AZURE_TENANT_ID="<Tenant>"
...
 
-# Azure cloud settings
-# To use the default public cloud, otherwise set to AzureChinaCloud|AzureGermanCloud|AzureUSGovernmentCloud
-export AZURE_ENVIRONMENT="AzurePublicCloud"
+# Settings needed for AzureClusterIdentity used by the AzureCluster
+export AZURE_CLUSTER_IDENTITY_SECRET_NAME="cluster-identity-secret"
+export CLUSTER_IDENTITY_NAME="cluster-identity"
+export AZURE_CLUSTER_IDENTITY_SECRET_NAMESPACE="default"
 
 export AZURE_SUBSCRIPTION_ID_B64="$(echo -n "$AZURE_SUBSCRIPTION_ID" | base64 | tr -d '\n')"
```
Comment (CecileRobertMichon) [inline docs/book/src/user/quick-start.md:218]: Do we still need the b64 encoded vars?

Comment (nader-ziada) [reply docs/book/src/user/quick-start.md:218]: its still in the tilt file so wasn't 100% sure, will try again now without them

Comment (nader-ziada) [reply docs/book/src/user/quick-start.md]: removed them

File: docs/book/src/user/quick-start.md [hunk 2]
```diff
@@ -210,14 +210,14 @@ export AZURE_TENANT_ID="<Tenant>"
...
-export AZURE_SUBSCRIPTION_ID_B64="$(echo -n "$AZURE_SUBSCRIPTION_ID" | base64 | tr -d '\n')"
-export AZURE_TENANT_ID_B64="$(echo -n "$AZURE_TENANT_ID" | base64 | tr -d '\n')"
-export AZURE_CLIENT_ID_B64="$(echo -n "$AZURE_CLIENT_ID" | base64 | tr -d '\n')"
-export AZURE_CLIENT_SECRET_B64="$(echo -n "$AZURE_CLIENT_SECRET" | base64 | tr -d '\n')"
+# Settings needed for AzureClusterIdentity used by the AzureCluster
+export AZURE_CLUSTER_IDENTITY_SECRET_NAME="cluster-identity-secret"
+export CLUSTER_IDENTITY_NAME="cluster-identity"
+export AZURE_CLUSTER_IDENTITY_SECRET_NAMESPACE="default"
+
+# Create a secret to incldue the password of the Service Principal identity created in Azure
```
Comment (CecileRobertMichon) [inline docs/book/src/user/quick-start.md:218]:
```suggestion
# Create a secret to include the password of the Service Principal identity created in Azure
```

Comment (nader-ziada) [reply docs/book/src/user/quick-start.md]: fixed, sorry

**Pull request merged**